packages/location_background/android/src/main/java/com/flutter/example/locationbackgroundplugin/LocationBackgroundPlugin.java

```diff
@@ -0,0 +1,22 @@
+package com.flutter.example.locationbackgroundplugin;
+
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel;
+import io.flutter.plugin.common.MethodChannel.MethodCallHandler;
+import io.flutter.plugin.common.MethodChannel.Result;
+import io.flutter.plugin.common.PluginRegistry.Registrar;
+
+/** LocationBackgroundPlugin */
+public class LocationBackgroundPlugin implements MethodCallHandler {
+  /** Plugin registration. */
+  public static void registerWith(Registrar registrar) {
+    final MethodChannel channel =
+        new MethodChannel(registrar.messenger(), "location_background_plugin");
+    channel.setMethodCallHandler(new LocationBackgroundPlugin());
+  }
+
+  @Override
+  public void onMethodCall(MethodCall call, Result result) {
+    result.notImplemented();
```

Comment:
If you had plans to implement the Android side, you might add a TODO here that points to the API to use and explains the strategy a bit.

Reply:
Done.